Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -444,10 +444,18 @@ public DisconnectedException(string message)
                     return "Cannot attach to debugger when already attached.";
                 }
 
+                var pyService = _eval._serviceProvider.GetPythonToolsService();
+
+                // Only pass options that make sense for the REPL.
+                var debugOptions = PythonDebugOptions.None;
+                if (pyService.DebuggerOptions.DebugStdLib) {
+                    debugOptions |= PythonDebugOptions.DebugStdLib;

```

Comment:
Could future options be interesting to pass.  If so add a comment to PythonDebugOptions letting developers know to update this location when they add a new option